woocommerce/woocommerce
# Skip metrics workflow for unrelevant changes

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





The Metrics workflow takes around 20 minutes to run on every change, many times not relevant to check the performance.
This PR adds some paths to be ignored to skip the workflow. It should help with freeing up runners.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create another PR from a branch having this one as base.
2. Update any file from the ignored paths
3. Check the CI. The metrics workflow should not run.
Tested with #45163



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
